packages/react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -458,7 +475,7 @@ class Dropdown extends AutoControlledComponent<Extendable<DropdownProps>, Dropdo
   private renderItemsList(
     styles: ComponentSlotStylesInput,
     variables: ComponentVariablesInput,
-    isOpen: boolean,
+    open: boolean,
```

Comment:
We can remove `open` from method params because we can get it from `this.state` now 🐰

Reply:
done, good catch